Testsuite/test/run_testsuite_with_cmake

```diff
@@ -42,9 +42,9 @@ process_tree()
   echo $pid
   ps -a | awk '!/^ +PID/ {print $1 " " $2}' | {
       while read apid appid; do
-	  if [ "$appid" = "$pid" ]; then
-	      process_tree $apid
-	  fi
+    if [ "$appid" = "$pid" ]; then
+        process_tree $apid
+    fi
```

Comment:
The indentation is now broken. If you cannot control correctly that feature of your editor, then disable it. It is really annoying, and something it leads to incorrect indentation. Historically, the width of a <kbd>TAB</kbd> was 8 spaces.